package.json

```diff
@@ -70,20 +71,22 @@
     "uuid": "9.0.0",
     "vue": "^3.1.0",
     "vue-eslint-parser": "9.3.1",
-    "webpack": "5.88.0",
     "vue-loader": "^16.0.0",
+    "webpack": "5.88.0",
     "webpack-cli": "5.1.1",
     "webpack-dev-server": "4.15.1",
     "webpack-merge": "5.9.0"
   },
   "scripts": {
     "clean": "rm -rf ./dist ./node_modules ./package-lock.json ./coverage ./html-test-results ./test-results ./.nyc_output ",
-    "clean-test-lint": "npm run clean; npm install; npm run test; npm run lint; npm run lint:spelling",
+    "clean-test-lint": "npm run clean; npm install; npm run test; npm run lint",
     "start": "npx webpack serve --config ./.webpack/webpack.dev.js",
     "start:coverage": "npx webpack serve --config ./.webpack/webpack.coverage.js",
-    "lint": "eslint example src e2e --ext .js openmct.js --max-warnings=0 && eslint example src --ext .vue",
+    "lint:js": "eslint example src e2e --ext .js openmct.js --max-warnings=0",
+    "lint:vue": "eslint example src --ext .vue",
     "lint:spelling": "cspell \"**/*.{js,md,vue}\" --show-context --gitignore",
-    "lint:fix": "eslint example src e2e --ext .js,.vue openmct.js --fix",
+    "lint": "run-p \"lint:* -- {1}\" --",
+    "fix:lint": "eslint example src e2e --ext .js,.vue openmct.js --fix",
```

Comment:
@evenstensberg after some internal discussion, we think `fix:lint` should be reverted to `lint:fix`

We will then need to explicitly list the scripts in `lint` as

`"lint":"run-p lint:js lint:vue lint:spelling"`

Lastly, it appears that `npm-run-all` is no longer maintained and we will want to switch to `npm-run-all2`